README.md

```diff
@@ -18,47 +16,62 @@
+
+> [!IMPORTANT]
+> [Subscribe Now](https://logto.io/subscribe/?utm_source=github&utm_medium=repo_logto) to stay updated with the latest information about the Logto and receive feature updates in real-time.
+
+## Key Features
 
-🧑‍💻 **Comprehensive frontend-to-backend identity solution**
+### 🧑‍💻 Comprehensive frontend-to-backend identity solution
 
-- Enables OIDC-based authentication with Logto SDKs.
+- Enables OpenID Connect (OIDC) based authentication with Logto SDKs.
 - Supports passwordless sign-in, along with various options like email, phone number, username, Google, Facebook, and other social sign-in methods.
 - Offers beautiful UI components with customizable CSS to suit your business needs.
-- Has an open community with many warm-hearted contributors and users. Check out our [awesome list](./AWESOME.md) of community-contributed resources.
 
-📦 **Out-of-the-box infrastructure**
+### 📦 Out-of-the-box infrastructure
 
-- Includes a ready-to-use Management API, serving as your authentication provider, thus eliminating the need for extra implementation.
+- Includes a ready-to-use [Management API](https://openapi.logto.io/), serving as your authentication provider, thus eliminating the need for extra implementation.
 - Provides SDKs that seamlessly integrate your apps with Logto across multiple platforms and languages, tailored to your development environment.
 - Offers flexible connectors that can be scaled with community contributions and customized with SAML, OAuth, and OIDC protocols.
 
-💻 **Enterprise-ready solutions**
+### 💻 Enterprise-ready solutions
 
-- Implements role-based access control (RBAC) for scalable role authorization, catering to a wide range of use cases.
-- Facilitates user management and provides audit logs for understanding identity-related user information and maintaining security.
-- Enables single sign-on (SSO) and multi-factor authentication (MFA) without extra coding.
-- Leverages Logto organizations to build multi-tenancy apps with ease.
+- Implements [Role-Based Access Control (RBAC)](https://docs.logto.io/docs/recipes/rbac/) for scalable authorization.
+- Enables user management with [audit logs](https://docs.logto.io/docs/recipes/inspect-audit-logs/) to track identity-related activities and maintain security.
+- Enables [single sign-on (SSO)](https://docs.logto.io/docs/recipes/single-sign-on/) and [multi-factor authentication (MFA)](https://docs.logto.io/docs/recipes/multi-factor-auth/) without minimal coding.
+- Leverage [Logto organizations](https://docs.logto.io/docs/recipes/organizations/understand-how-it-works/) to build [multi-tenancy](https://blog.logto.io/tenancy-models/) apps with ease.
 
-In a more approachable way, we refer to this solution as "[Customer Identity Access Management (CIAM)](https://en.wikipedia.org/wiki/Customer_identity_access_management)" or simply, the "Customer Identity Solution."
+## Customer Identity Access Management (CIAM) Introductory Courses
```

Comment:
I think we prefer some of these secondary titles to be in sentence case instead.

Reply:
H2 and H3 styled in sentence case